scene/2d/animated_sprite.cpp

```diff
@@ -490,6 +490,34 @@ Point2 AnimatedSprite::get_offset() const {
 	return offset;
 }
 
+void AnimatedSprite::set_anchor(const Point2 &p_anchor) {
+
+	anchor = p_anchor;
+	anchor.x = CLAMP(anchor.x, -1, 1);
+	anchor.y = CLAMP(anchor.y, -1, 1);
+
+	Point2 calc_anchor = anchor;
+
+	if (centered) {
+		calc_anchor.x = CLAMP(anchor.x, -0.5, 0.5);
+		calc_anchor.y = CLAMP(anchor.y, -0.5, 0.5);
+	}
+
+	Size2 s;
+	Ref<Texture> texture = frames->get_frame(animation, frame);
+	s = Size2(texture->get_size());
+
+	offset = Point2(s.width * calc_anchor.x, s.height * -calc_anchor.y);
```

Comment:
Makes sense. Should I add offset to the calculated anchor value? So it's anchor/normalized-offset + offset?